agent/version/version.go

```diff
@@ -22,10 +22,10 @@ package version
 // repository. Only the 'Version' const should change in checked-in source code
 
 // Version is the version of the Agent
-const Version = "1.82.3"
+const Version = "1.82.4"
 
 // GitDirty indicates the cleanliness of the git repo when this agent was built
 const GitDirty = true
 
 // GitShortHash is the short hash of this agent build
-const GitShortHash = "99c5e65a"
+const GitShortHash = "53e8e5ec"
```

Comment:
We might need to update the short sha to be  the one for this [commit](https://github.com/aws/amazon-ecs-agent/commit/428c2f40602927902ff04fbdb3d69dc968466a45) `428c2f40`.  Perhaps try rebasing and re-running the generation scripts again?

Reply:
Set short hash to `5847274a`